src/traces/choropleth/index.js

```diff
@@ -9,9 +9,9 @@
 
 'use strict';
 
-var Plotly = require('../../plotly');
+var Plots = require('../../plots/plots');
```

Comment:
`Plots` can stand on its own now so we should change things to directly require it whenever we come across `Plotly.Plots`.